app/views/travel_insurance_registrations/medical_conditions_questionaire_form.html.erb

```diff
@@ -0,0 +1,73 @@
+<%= heading_tag(t(registration_title), level: 1) %>
+<p><%= t('travel_insurance_registrations.intro') %></p>
+
+<h3>Step 4 of 4 </h3>
```

Comment:
I don't think, semantically-speaking, this is a heading. I'd prefer to see it marked up as a `<p>` with a classname that reflects its role on the page. If it is a heading it should be a `<h2>` as to skip the numerical ordering of headings can be confusing to assistive technology.

Reply:
Updated to use a `p` tag